### PR TITLE
Book detail

### PR DESCRIPTION
**Pull request by CristianRWolox**
## Summary

- Added layout of bookDetail.

#### Galaxy S (or similar)
image

#### Pixel 2/3 (or similar)
image

#### Pixel 3XL (or similar)
image

#### iPhone SE (or similar)
image

#### iPhone 6/7/8 (or similar)
image

#### iPhone 11/12 Base/Pro/Pro Max (or similar)
simulator_screenshot_69BB0C32-39AB-4C8A-8225-1AC61F13658F

## Trello Card

https://trello.com/c/FZ7dULZj/17-n-72-detalle-del-libro

### CODE REVIEW

**Review by mcolombo87 — COMMENTED** [review]
The "into <Branch>" must be **navigation**. I can see changes in this PR that aren't of this card.

File: wbooks/src/app/screens/BookDetail/components/Information/index.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const Information = ({ title, author, imageUrl, year, genre }: Props) => {
+  return (
+    <View style={styles.bookCard}>
+      <View style={styles.dataContainer}>
+        <Image source={imageUrl ? { uri: imageUrl } : bcInicio} style={styles.detailImageBook} />
+        <View style={styles.textContainer}>
+          <Text style={styles.title}>{title}</Text>
+          <Text style={styles.author}>{author}</Text>
+          <Text style={styles.author}>{year}</Text>
+          <Text style={styles.author}>{genre}</Text>
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/Information/index.tsx]: use a better common name

File: wbooks/src/app/screens/BookDetail/components/Information/styles.ts [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    color: COLORS.primary
+  },
+  btnRent: {
+    alignSelf: 'center',
+    alignItems: 'center',
+    marginTop: 20,
+    paddingVertical: 12,
+    borderRadius: 30,
+    width: '90%',
+    backgroundColor: COLORS.primary
```
Comment (guidoprinc) [inline wbooks/src/app/screens/BookDetail/components/Information/styles.ts]: many of this styles are repeated in btnAdd. Extract them as a common style

**Review by guidoprinc — CHANGES_REQUESTED** [review]

File: wbooks/src/app/screens/BookDetail/components/Information/index.tsx (hunk 1 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/BookDetail/components/Information/index.tsx]: Done!

File: wbooks/src/app/screens/BookDetail/components/Information/styles.ts (hunk 2 above)
Comment (CristianRWolox) [reply wbooks/src/app/screens/BookDetail/components/Information/styles.ts]: Done!

**Review by guidoprinc — APPROVED** [review]

File: wbooks/src/app/components/Button/index.tsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+import { TouchableOpacity, Text } from 'react-native';
+
+import styles from './styles';
+
+interface Props {
+  solid?: boolean;
+  text: string;
+}
+
+const Information = ({ solid = true, text }: Props) => {
```
Comment (mcolombo87) [inline wbooks/src/app/components/Button/index.tsx:11]: Information?? Button, CustomButton perhaps?

**Pull request merged**